PHPUnit/Extensions/Selenium2TestCase/Driver.php

```diff
@@ -132,6 +132,9 @@ public function curl($http_method,
         if ($info['http_code'] == 404) {
             throw new BadMethodCallException("The command $url is not recognized by the server.");
         }
+        if (($info['http_code'] > 399) && ($info['http_code'] < 500)) {
```

Comment:
I think the `[400, 500)` interval is fine though. Just don't like the non-round `399`